com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/transaction/ZigBeeTransactionManager.java

```diff
@@ -214,6 +215,7 @@ public void run() {
                 });
             }
         }
+        executorService.shutdownNow();
```

Comment:
This has been done at the start of this method already.

Reply:
Are you sure? We have `shutdown` - this marks the executor so it doesn't accept any further requests, but it will not terminate the queue. I don't see `shutdownNow` though?

Reply:
Oh right, I saw the `executorService.shutdown()` and thought its `executorService.shutdownNow()`.

All good then. Sorry for the noise.